CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.10.1dev
 * [Feature] Automatically connect if the `dsn_filename` (defaults to `~/.jupysql/connections.ini`) contains a `default` section
+* [Fix] Robust statement type checking in resultset ([#794])
```

Comment:
the changelog entry should only include the number within parenthesis (those square brackets shouldn't  be there)

Reply:
[Fixed!](https://github.com/ploomber/jupysql/pull/797/commits/b00b8e395e34aa0ca5050fbf5f2d167bd40ca26c)